src/scripts/sync_teams/sync_teams.py

```diff
@@ -174,11 +182,33 @@ def process_teams(token, organization, assets: List[Asset]) -> List[str]:
     teams_to_create = get_teams_to_create(desired_teams, existing_team_names)
     teams_to_delete = get_teams_to_delete(desired_teams, existing_team_names)
     if teams_to_create:
-        logger.info(f"Creating {len(teams_to_create)} team(s): {teams_to_create}")
+        logger.info(
+            f"Creating {len(teams_to_create)} team(s): {teams_to_create}")
         create_teams(token, teams_to_create)
     return teams_to_delete
 
 
+def process_members(token: str, organization: Organization, existing_teams: List[TeamAttributes]) -> None:
+    logger.info("Processing team members.")
+    for team_structure in organization.teams:
+        team_name = team_structure.name
+        team_members = team_structure.members
+
+        # Find the corresponding existing team
+        existing_team = next(
+            (team for team in existing_teams if team.name == team_name), None)
```

Comment:
not sure what's the scale of teams here, but this can be inefficient in case of scale..

Reply:
Given that it's an in-memory search, I doubt we'll have an issue here even with 1000s of teams. We can discuss it if you disagree.

Reply:
Given that it’s an in-memory search, I doubt we’ll have an issue here even with 1000s of teams. We can discuss it if you disagree.